src/lib/fetch-amis-for-address.ts

```diff
@@ -17,6 +17,25 @@ export default async function fetchAMIsForAddress(
 
   const zip = result.address_components.zip;
 
+  // We pull these from our database (vs geocoder) to ensure a match
+  // when computing whether local incentives are eligible.
+  const supplemental = await db.get<GeoInfo>(
+    `
+    SELECT 
+        city, 
+        county_name as county
+    FROM zips
+    WHERE zip = ?
+  `,
+    zip,
+  );
```

Comment:
We've run straight into the limits of our zip code data here -- this table has one row per zip, but there are plenty of zips that span city and county borders. (E.g. 12303 -- the data says it's Albany County, NY but a good chunk of it is in Schenectady County too.)

There are at least two threads to pull on:

- Getting hold of a many-to-many mapping between zip codes and cities/counties.
- Figuring out what to do when a user enters a zip that spans cities/counties. Do we return _no_ city/county incentives? Do we return the _union_ of possible city/county incentives? Do we have to introduce some kind of "maybe eligible depending on your exact location" status? (The latter idea has been around for a while, but we've never been forced to actually confront it yet.)

Reply:
I think we want better treatment of this to block state calculator launches, but not beta testing. It's important for us to be able to see local incentives sooner rather than later (on Tuesday as currently scheduled). This is the direction I got from Tom when chatting with him on Slack (he's too busy for a more substantive review).

I've added a number of comments indicating this is a limited solution, and I filed https://app.asana.com/0/1204738794846444/1206454407609847 to track the work for getting to a better long-term solution, but can we proceed with this as a stopgap for now?